gradle/enforcement/checkstyle.xml

```diff
@@ -262,7 +262,7 @@
     <module name="AbbreviationAsWordInName">
       <property name="ignoreFinal" value="false"/>
       <!-- modified from default to support KHttp -->
-      <property name="allowedAbbreviationLength" value="1"/>
+      <property name="allowedAbbreviationLength" value="10"/>
```

Comment:
can this be changed back? it looks like the issue was the uppercase instance field `APP` that you fixed already
```suggestion
      <property name="allowedAbbreviationLength" value="1"/>
```